collector/mongod/replset_status.go

```diff
@@ -1,232 +1,223 @@
 package collector_mongod
 
 import (
-    "time"
-    "github.com/golang/glog"
-    "github.com/prometheus/client_golang/prometheus"
-    "gopkg.in/mgo.v2"
-    "gopkg.in/mgo.v2/bson"
+	"time"
+
```

Comment:
replset_status.go is an improved version of my code pulled down from mainline, so this was a choice from the upstream committer, unfortunately. It can be removed